asserts/snap_asserts.go

```diff
@@ -194,6 +197,12 @@ func snapDeclarationFormatAnalyze(headers map[string]interface{}, body []byte) (
 		if rule.feature(nameConstraintsFeature) {
 			setFormatNum(4)
 		}
+		if rule.feature(nameConstraintsFeature) {
+			setFormatNum(4)
+		}
```

Comment:
is this meant to be duplicated with the above lines?

Reply:
this is a mistaken repetition